components/hab/Cargo.toml

```diff
@@ -17,7 +17,7 @@ base64 = "*"
 bitflags = "*"
 chrono = {version = "*", features = ["serde"]}
 clap = { git = "https://github.com/habitat-sh/clap.git", branch = "v2-master", features = [ "suggestions", "color", "unstable" ] }
-configopt = { git = "https://github.com/davidMcneil/configopt.git" }
+configopt = { git = "https://github.com/pozsgaic/configopt.git" }
```

Comment:
we should fork this into the habitat-sh org.

Reply:
I forked to habitat-sh and renamed master to main in the process